test/common/http/conn_manager_impl_test_base.h

```diff
@@ -235,6 +236,7 @@ class HttpConnectionManagerImplMixin : public ConnectionManagerConfig {
   HttpConnectionManagerProto::ServerHeaderTransformation server_transformation_{
       HttpConnectionManagerProto::OVERWRITE};
   absl::optional<std::string> scheme_;
+  bool scheme_match_upstream_{false};
```

Comment:
nit: If this is not being overwritten, consider converting to const.

Reply:
Need this to be non-const to override in the `PassMatchUpstreamSchemeHintToStreamInfo` test I wrote.